tailwind.config.js

```diff
@@ -57,6 +57,9 @@ module.exports = {
         "vh-1/3": "calc(100vh / 3)",
         "vh-1/4": "calc(100vh / 4)",
         "vh-1/5": "calc(100vh / 5)",
+      },
+      minWidth: {
```

Comment:
I don't think you need to add anything special. Tailwind has min-width utils already